CHANGELOG.mdx

```diff
@@ -28,7 +28,7 @@ https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
 
 - [#6044](https://github.com/rasahq/rasa/issues/6044>): Do not deepcopy slots when instantiating trackers. This leads to a significant
   speedup when training on domains with a large number of slots.
-- [#6226](https://github.com/rasahq/rasa/issues/6226>): Added more debugging logs to the [Lock Stores](./lock-stores) to simplify debugging in case of
+- [#6226](https://github.com/rasahq/rasa/issues/6226>): Added more debugging logs to the [Lock Stores](./lock-stores.mdx) to simplify debugging in case of
```

Comment:
@erohmensing I've noticed that some items in the changelog are not formatted correctly, for instance here:
```suggestion
- [#6226](https://github.com/rasahq/rasa/issues/6226): Added more debugging logs to the [Lock Stores](./lock-stores.mdx) to simplify debugging in case of

```

Reply:
ooh there seems to be a few more of those (see 6044 above it) -- i think remnants from my removing the anchors. good catch